.changelog/2949.bugfix.md

```diff
@@ -0,0 +1,6 @@
+go/oasis-node: Fix signer configuration via YAML
+
+The `--signer` argument has been renamed to `--signer.backend` to allow
```

Comment:
Wondering if it would be useful to have a "configuration change" fragment type? (iirc currently we just have the label)

Reply:
Good idea, I'll open an issue.